src/lib/striptags.js

```diff
@@ -24,223 +24,210 @@
 
 'use strict';
 
-(function(global) {
```

Comment:
this is copy pasted from somewhere?

Reply:
That piece of code aims to support different module systems. Our setup started to complain about it, so I simplified it. No functionality lost.